container/single-instance-codespaces/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: MIT
+#!BuildTag: openqa-single-instance-codespaces:latest opensuse/openqa-single-instance-codespaces:latest opensuse/openqa-single-instance-codespaces:%PKG_VERSION% opensuse/openqa-single-instance-codespaces:%PKG_VERSION%.%RELEASE%
+
+# hadolint ignore=DL3006
+FROM openqa-single-instance
```

Comment:
isnt that a Leap? I thought that it should be a TW. should we have a comment? wdyt?

Reply:
It is Leap right now, but it will be Tumbleweed in a near future, see #5819.

Reply:
I just saw https://github.com/os-autoinst/openQA/pull/5819/files#diff-cd62a72c6c09ba757d9d23c56722b50b7c1b2e2da6a7a0c496c8a8e6e3ce8e9fR5. Still it would be nice to have a link to the container/single-instance/Dockerfile here?